hw/ip/prim/prim_secded.core

```diff
@@ -14,6 +14,8 @@ filesets:
       - rtl/prim_secded_28_22_enc.sv
       - rtl/prim_secded_39_32_dec.sv
       - rtl/prim_secded_39_32_enc.sv
+      - rtl/prim_secded_64_57_dec.sv
+      - rtl/prim_secded_64_57_enc.sv
```

Comment:
since we're adding a new primitive here, can you also check in the generated FPV testbench for this primitive?

Reply:
added